docs/how to/use-transferbench.rst

```diff
@@ -36,71 +39,102 @@ This simplifies to a simple copy operation when dealing with single SRC/DST.
+  Transfers (srcMem1->Executor1->dstMem1 SEs1 Bytes1) ... (srcMemL->ExecutorL->dstMemL SEsL BytesL)
 
-   1 4 (G0->G0->G1)                   Uses 4 CUs on GPU0 to copy from GPU0 to GPU1
-   1 4 (C1->G2->G0)                   Uses 4 CUs on GPU2 to copy from CPU1 to GPU0
-   2 4 G0->G0->G1 G1->G1->G0          Copes from GPU0 to GPU1, and GPU1 to GPU0, each with 4 SEs
-   -2 (G0 G0 G1 4 1M) (G1 G1 G0 2 2M) Copies 1Mb from GPU0 to GPU1 with 4 SEs, and 2Mb from GPU1 to GPU0 with 2 SEs
+  The arguments used to specify transfers in the config file are described in the :ref:`arguments table <config_file_arguments_table>`.
+
+  **Example**:
+
+  .. code-block:: bash
+
+   -2 (G0 G0 G1 4 1M) (G1 G1 G0 2 2M) Copies 1Mb from GPU0 to GPU1 with 4 SEs and 2Mb from GPU1 to GPU0 with 2 SEs
+
+Here is the list of arguments used to specify transfers in the config file:
+
+.. _config_file_arguments_table:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Argument
+     - Description
+
+   * - Transfers
+     - Number of transfers to be run in parallel
+
+   * - SE
+     - Number of SEs to use (CPU threads or GPU threadblocks)
+
+   * - srcMemL
+     - Source memory locations (where the data is read)
+
+   * - Executor
+     - | Executor is specified by a character indicating type, followed by the device index (0-indexed):
+       | - C: CPU-executed  (indexed from 0 to NUMA nodes - 1)
+       | - G: GPU-executed  (indexed from 0 to GPUs - 1)
+       | - D: DMA-executor  (indexed from 0 to GPUs - 1)
+
+   * - dstMemL
+     - Destination memory locations (where the data is written)
+
+   * - bytesL
+     - | Number of bytes to copy (use command-line specified size when 0).
+       | Must be a multiple of 4 and may be suffixed with ('K','M', or 'G').
+       | Memory locations are specified by one or more device characters or device index pairs.
+       | Characters indicate memory type and are followed by device index (0-indexed).
+       | Here are the characters and their respective memory locations:
+       | - C:    Pinned host memory       (on NUMA node, indexed from 0 to [NUMA nodes-1])
+       | - U:    Unpinned host memory     (on NUMA node, indexed from 0 to [NUMA nodes-1])
+       | - B:    Fine-grain host memory   (on NUMA node, indexed from 0 to [NUMA nodes-1])
+       | - G:    Global device memory     (on GPU device, indexed from 0 to [GPUs - 1])
+       | - F:    Fine-grain device memory (on GPU device, indexed from 0 to [GPUs - 1])
+       | - N:    Null memory              (index ignored)
 
-Round brackets and arrows' ->' may be included for human clarity, but will be ignored and are unnecessary
-Lines starting with will be ignored. Lines starting with will be echoed to output
+Round brackets and arrows "->" can be included for human clarity, but will be ignored.
+Lines starting with # will be ignored. Lines starting with ## will be echoed to output.
 
-Single GPU-executed Transfer between GPUs 0 and 1 using 4 CUs::
+**Transfer examples:**
```

Comment:
There is inconsistency in the space between GPU and the number in this section. Review all of them. 
![image](https://github.com/user-attachments/assets/c41a7181-0d20-4d09-a437-b2aa946a4980)

Reply:
Corrected.